src/components/KuadrantOverviewPage.tsx

```diff
@@ -384,7 +384,9 @@ const KuadrantOverviewPage: React.FC = () => {
   ): Record<string, Distribution> => {
     const key = `${obj.metadata.namespace}/${obj.metadata.name}-istio`;
     const codes = totalRequestsByGateway[key]?.codes ?? {};
-    const filteredCodes = Object.entries(codes).filter(([code]) => code.startsWith(prefix));
+    const filteredCodes = Object.entries(codes)
+      .filter(([code]) => code.startsWith(prefix))
+      .sort(([, count1], [, count2]) => count2 - count1);
```

Comment:
the sorting isnt showing in decending order 
![Screenshot 2025-04-08 at 15 19 54](https://github.com/user-attachments/assets/c59c81d3-c005-4ade-902b-0a4d0df29404)